src/workspace/Lesson/Header/index.jsx

```diff
@@ -38,21 +39,27 @@ const Header = () => {
     const { store, dispatch } = useContext(LessonContext)
     const [showSaveStatus, setShowSaveStatus] = useState(false)
 
+    const {
```

Comment:
Sollte LessonContext von Naming bleiben? Es ist an sich nicht der lessonContext, sondern alle contexte.

Reply:
na irgendwie schon, ist ja lesson spezifisch, wird vermutlich dann halt noch einen Homework kontext geben, aber sich wird die combination von reducern nach dem context gewählt, auch wenn eigentlich alle bist auf lesson auch in den anderen sein dürften...